freqtrade/optimize/__init__.py

```diff
@@ -1,49 +1 @@
-# pragma pylint: disable=missing-docstring
-
-import logging
-from datetime import datetime
-from typing import Dict, Tuple
-import operator
-
-import arrow
-from pandas import DataFrame
-
 from freqtrade.optimize.default_hyperopt import DefaultHyperOpts  # noqa: F401
-
```

Comment:
<s>If this file is really needed if it's almost empty?</s>
Aha, it's not empty in the next commit, forget it (reviewing here commit by commit, not the cumulative results...)